test/system/src/test/java/io/pravega/test/system/MetadataScalabilityTest.java

```diff
@@ -47,24 +46,14 @@
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.stream.Collectors;
 
-/**
- * This test creates a stream with 10k segments and then rapidly scales it 1010 times.
- * Then it performs truncation a random number (less than 1010) of times. 
- */
 @Slf4j
 @RunWith(SystemTestRunner.class)
-public class MetadataScalabilityTest extends AbstractScaleTests {
-    private static final String STREAM_NAME = "metadataScalability";
-    private static final int NUM_SEGMENTS = 10000;
-    private static final StreamConfiguration CONFIG = StreamConfiguration.builder()
-                                                                         .scalingPolicy(ScalingPolicy.fixed(NUM_SEGMENTS)).build();
-    private static final int SCALES_TO_PERFORM = 1010;
-
+public abstract class MetadataScalabilityTest extends AbstractScaleTests {
     @Rule
     public Timeout globalTimeout = Timeout.seconds(60 * 60);
```

Comment:
Are these tests going to take up to 1 hour each? We need to think to the system test build time as well.

Reply:
Observation is that it takes about 25 minutes